drivers/ethernet/dsa_ksz8xxx.c

```diff
@@ -1016,14 +1067,13 @@ static enum ethernet_hw_caps dsa_port_get_capabilities(const struct device *dev)
 {
 	ARG_UNUSED(dev);
 
-	return ETHERNET_DSA_SLAVE_PORT | ETHERNET_LINK_10BASE_T |
-		ETHERNET_LINK_100BASE_T;
+	return ETHERNET_DSA_SLAVE_PORT | ETHERNET_LINK_10BASE_T | ETHERNET_LINK_100BASE_T;
 }
 
 const struct ethernet_api dsa_eth_api_funcs = {
-	.iface_api.init		= dsa_iface_init,
-	.get_capabilities	= dsa_port_get_capabilities,
-	.send                   = dsa_tx,
+	.iface_api.init = dsa_iface_init,
+	.get_capabilities = dsa_port_get_capabilities,
+	.send = dsa_tx,
```

Comment:
Basically all of these changes are cosmetic and do not belong to this commit. I am seeing similar changes all over this PR. If you want to tweak the look and feel, please do the changes in separate commit (could be in this PR) so that the actual changes that introduce the new driver can be clearly seen.

Reply:
@jukkar Cosmetic changes are the result of clang-format's work. Ok, I removed them.